fc/language/expressions/abstract.py

```diff
@@ -155,17 +123,17 @@ def GetDefiningEnvironments(self, env):
         """Cache which environment each variable used in this expression is actually defined in.
 
         Stores each environment indexed by the local name of the variable within that environment.
-        
+
         TODO: Handle local name conflicts!
         """
         try:
             return self._definingEnvs
         except AttributeError:
             self._rootDefiningEnv = env  # For paranoia checking that the cache is valid
             d = self._definingEnvs = {}
-            l = self._usedVarLocalNames = []
+            l = self._usedVarLocalNames = []  # noqa: E741
```

Comment:
Should we disable this one project-wide?

Reply:
Not sure. It's generally a sensible warning, except for very small bits of code like this where there's no chance of confusion!